src/spn/algorithms/LearningWrappers.py

```diff
@@ -113,6 +113,7 @@ def learn_mspn(
     rows="kmeans",
     min_instances_slice=200,
     threshold=0.3,
+    initial_scope=None,
```

Comment:
What is the need to have an initial scope?

Reply:
For SPMN implementation, we iterate through a partial order of variables and call these functions in each iteration and merge the spns learned. For example, let `V` represent R.V's and` D `represent Decision variables, and  `[[V0,V1], [D0],[V2,V3,V4], [V5,V6]]` be the partial order, so we learn spns on `[V0,V1];scope=[0,1]`, next on [`V2,V3,V4]; scope[2,3,4]` and so on. Hence, the need for initial scope.